docs/installation/deployment/kubernetes.rst

```diff
@@ -0,0 +1,280 @@
+=======================
+Deploying on Kubernetes
+=======================
+
+.. epigraph::
+
+    `Kubernetes`_ is an open-source system for automating deployment,
+    scaling, and management of containerized applications.
+
+    -- kubernetes.io
+
+
+Kubernetes (K8s) is one of the main supported targets for deployment. We
+provide a fully automated deployment toolset suitable for DevOps engineers
+and/or system administrators.
+
+This documentation will sketch out the architecture, document the initial
+requirements and teach you how to deploy on K8s.
+
+This setup is tested against a Google Cloud Kubernetes cluster with 4vCPU and
+15G of memory.
+
+Application architecture
+========================
+
+Typically, your K8s cluster provider will set up a load balancer in front
+of your cluster, directing all the traffic to services in the cluster.
+
+DNS is set up so that your domain name(s) point to the IP address of the load
+balancer.
+
+The load balancer will then pass the traffic to an *Ingress* in the cluster,
```

Comment:
Add link to Ingress

Reply:
Mentioned the concept in the intro paragraph now with a link to the k8s docs